docs/integrations/sources/hubspot.md

```diff
@@ -69,6 +70,29 @@ Hubspot's API will [rate limit](https://developers.hubspot.com/docs/api/usage-de
 This connector supports only authentication with API Key. To obtain API key for the account go to settings -&gt; integrations \(under the account banner\) -&gt; api key. If you already have an api key you can use that. Otherwise generated a new one. See [docs](https://knowledge.hubspot.com/integrations/how-do-i-get-my-hubspot-api-key) for more details.
 
 
+### Required scopes
+
+Most of the streams require the [scopes](https://legacydocs.hubspot.com/docs/methods/oauth2/initiate-oauth-integration#scopes) enabled for the API account.
```

Comment:
isn't this only required if using Oauth2, which the connector currently doesn't support?

Reply:
Yes, it is. API key is getting the required scope automatically. 
Should we close this PR for now then?

Reply:
let's close it and add a note to the hubspot oauth issue saying to merge this PR once Hubspot Oauth is added